tutorials/getting_started/getting_started.ipynb

```diff
@@ -32,7 +31,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The next step is to initialize an `Experiment` instance. The `Experiment` must be provided a name. This can be any string, but it is best practice to give it a meaningful name as a broad title for what types of models the experiment will be supervising. For our purposes, our `Experiment` will be named `\"getting-started\"`.\n",
+    "The next step is to initialize an `Experiment` instance. The `Experiment` must be provided a name which can be any string, but it is best practice to give it a meaningful name as a broad title for what types of models the experiment will be supervising. For our purposes, our `Experiment` will be named `\"getting-started\"`.\n",
```

Comment:
need a comma after name if you're going to make this change, but you've built what is very nearly a runon sentence here. I think I like the old version better.  Maybe just say "The model name" instead of "This"?